mathesar/views.py

```diff
@@ -182,13 +182,14 @@ def get_common_data(request, database=None, schema=None):
     }
 
 
-def get_current_database(request, db_name):
+def get_current_database(request, connection_id):
     """Get database from passed name, with fall back behavior."""
     successful_dbs, failed_dbs = _get_permissible_db_queryset(request)
     permitted_databases = successful_dbs | failed_dbs
-    if db_name is not None:
-        current_database = get_object_or_404(permitted_databases, name=db_name)
+    if connection_id is not None:
+        current_database = get_object_or_404(permitted_databases, id=connection_id)
     else:
+        # TODO: refactor connection names to ids
```

Comment:
I do not think this will break the demo setup. I'd like @mathemancer to take a look and confirm once, post merge.

Reply:
@seancolsen I hope you'll followup regarding this.

Reply:
I removed the code comment in #3354.